book/website/collaboration/leadership/leadership-building.md

```diff
@@ -0,0 +1,103 @@
+Sometimes it may mean that they reach out to the people who don't actively engage, or feel left out of the consensus to make decisions that are best for the maximum number of people affected by it.
+- **Trust**: Healthy leadership trust in the skills and ability of people around them and identify the best fit for everyone to succeed.
+They are aware of their own biases and avoid them intentionally, often by providing space to listen to different thoughts and perspectives. 
+They guide people to adapt to changes and transformations particularly when leading large projects, with many moving parts and teams.
+- **Delegation**: Leaders are to lead by example, but they will be unable to do everything and should not do everything either. 
+When a leader is in charge of too many aspects of a project they will become a bottleneck and blocker to the emergence of other open leaders.
+Delegation goes hand in hand with building a trustworthy environment, this means that they communicate clearly the overarching goals, create smaller milestones and delegate tasks meaningfully so different people can lead different parts of the work, nurturing healthy leadership habits within their teams.
+- **Power dynamics**: Power dynamics are relationships between people in an organisation with varying levels of privileges. 
+Power dynamics are often created by leaders.
+Leaders with formal authority come with institutional power and open leaders often come with informal connections that can translate into social capital or power (even when they don’t have formal authority in the organisation).
+Therefore, leaders should understand how their positions affect the dynamics when working with others in their team, even when not exerting their power.
+Understanding power dynamics helps leaders identify and resolve conflict, build consensus, and create a more effective team culture.
+- **Acknowledging fairly**: Leadership roles come with a lot of visibility for their roles.
+This is partly earned by leaders' work as individuals but is often enhanced by the work of others in their teams. 
+Open leaders recognise and celebrate all kinds of work, appreciate and celebrate them openly and give appropriate credit to everyone, including those whose work makes them visible often taking invisible responsibilities for the sake of the overall success.
+
+##  What should Leaders Avoid
+
+- **Elitism and hypocrisy**: Open leadership in research in general benefits a lot from equity, inclusivity and diversity (EDI). 
+Thus, traits that sabotage EDI goals, should be avoided at all costs.
+Attitudes like elitism and hypocrisy can have unintended consequences on the project as well as on the leaders themselves.
+Elitism is when leaders constantly reinforce their authority, status, and expertise, creating distance between them and junior/new members of their teams.
+Hypocrisy is holding others accountable for one set of quality measures, and having a lesser standard for themselves.
```

Comment:
This is a very specific definition for quite a general phrase in English - we might want to consider if we need to be more specific in our phrasing (for example a lot of English speakers won't interpret hypocrisy as different standards automatically)

Reply:
How would you define it?

Reply:
```suggestion
Hypocrisy is when leaders hold their team members to a different standard than that to which they hold themselves.
```